agent/balrogagent/test/test_cmd.py

```diff
@@ -171,11 +171,11 @@ def side_effect(balrog_api_root, endpoint, auth, loop, method='GET'):
     async def testMultipleChangesOneEndpoint(self, time, time_is_ready, telemetry_is_ready, request):
         time.return_value = 999999999
         time_is_ready.return_value = True
-        sc = {'releases': [{"sc_id": 4, "when": 234, "telemetry_uptake": None,
+        sc = {'releases': [{"priority": None, "sc_id": 4, "when": 234, "telemetry_uptake": None,
                             "telemetry_product": None, "telemetry_channel": None},
-                           {"sc_id": 5, "when": 234, "telemetry_uptake": None,
+                           {"priority": None, "sc_id": 5, "when": 234, "telemetry_uptake": None,
                             "telemetry_product": None, "telemetry_channel": None},
-                           {"sc_id": 6, "when": 234, "telemetry_uptake": None,
+                           {"priority": None, "sc_id": 6, "when": 234, "telemetry_uptake": None,
                             "telemetry_product": None, "telemetry_channel": None}]}
```

Comment:
We need at least one test that has multiple different "priority" and "when" set, and we need to make sure that the "enact" endpoint is called in the right order. This will be similar to this test, but the order of the entries of "called_endpoints" will be important.